applications/StructuralMechanicsApplication/custom_python/structural_mechanics_python_application.cpp

```diff
@@ -188,6 +190,32 @@ BOOST_PYTHON_MODULE(KratosStructuralMechanicsApplication)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(NEGATIVE_FACE_PRESSURES_VECTOR )
 
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(INELASTIC_FLAG)
+
+    // Adjoint Variables
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(ADJOINT_DISPLACEMENT)
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(ADJOINT_ROTATION)
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(DISTURBANCE_MEASURE)
+
+    // Variables for output of sensitivities
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(CROSS_AREA_SENSITIVITY);
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS( POINT_LOAD_SENSITIVITY );
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( I22_SENSITIVITY );
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( I33_SENSITIVITY );
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( THICKNESS_SENSITIVITY );
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( YOUNG_MODULUS_SENSITIVITY);
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( AREA_EFFECTIVE_Y_SENSITIVITY);
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( AREA_EFFECTIVE_Z_SENSITIVITY);
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE( IS_ADJOINT );
```

Comment:
Better to use a flag for this

Reply:
how can he use a flag? it is not in the list of the base flags and i guess this should last through the analysis, so i think current usage is ok